src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -81,11 +84,11 @@ object Boot extends IOApp with LazyLogging {
 
         _ <- dependencies.cloudExtensionsInitializer.onBoot(dependencies.samApplication)
 
-        binding <- IO.fromFuture(IO(Http().newServerAt("0.0.0.0", 8080).bind(dependencies.samRoutes.route))).onError {
+        httpServer = IO.fromFuture(IO(Http().newServerAt("0.0.0.0", 8080).bind(dependencies.samRoutes.route))).onError {
           case t: Throwable => IO(logger.error("FATAL - failure starting http server", t)) *> IO.raiseError(t)
-        }
-        _ <- IO.fromFuture(IO(binding.whenTerminated))
-        _ <- IO(system.terminate())
+        }.void
```

Comment:
```
  /*
   * Ignores the result of this IO.
   */
  def void
```

(I'm a bit confused) isn't httpServer just IO[Unit]? Why would that be useful to pass into the next line?